libs/wire-api/src/Wire/API/User.hs

```diff
@@ -1208,6 +1213,28 @@ instance ToSchema VerificationAction where
           element "login" Login
         ]
 
+instance ToByteString VerificationAction where
+  builder GenerateScimToken = "generate_scim_token"
+  builder Login = "login"
+
+instance FromByteString VerificationAction where
+  parser =
+    Parser.takeByteString >>= \b ->
+      case T.decodeUtf8' b of
+        Right "login" -> pure Login
+        Right "generate_scim_token" -> pure GenerateScimToken
+        Right t -> fail $ "Invalid VerificationAction: " <> T.unpack t
+        Left e -> fail $ "Invalid VerificationAction: " <> show e
+
+instance S.ToParamSchema VerificationAction where
+  toParamSchema _ = S.toParamSchema (Proxy @Text)
```

Comment:
This should be an enum schema, not a free text, but I'm ok with being lazy, eg.;

```suggestion
  toParamSchema _ =
    mempty
      { S._paramSchemaType = Just S.SwaggerString,
        S._paramSchemaEnum = Just (A.String . toQueryParam <$> [(minBound :: VerificationAction) ..])
      }
```

On second thought, this shouldn't even be necessary, you should derive it via the schema-profunctor instance we already have.